types/ramda/index.d.ts

```diff
@@ -768,7 +771,12 @@ export function endsWith<T>(subList: readonly T[]): (list: readonly T[]) => bool
  */
 export function eqBy<T, U = T>(fn: (a: T) => U, a: T, b: T): boolean;
 export function eqBy<T, U = T>(fn: (a: T) => U, a: T): (b: T) => boolean;
-export function eqBy<T, U = T>(fn: (a: T) => U): _.F.Curry<(a: T, b: T) => boolean>;
+export function eqBy<T, U = T>(
```

Comment:
We are not care about fn's return type, maybe we can remove U, and rewrite `eqBy`'s type like this?:

```ts
export function eqBy<T>(fn: (a: T) => void, a: T, b: T): boolean;
```

Reply:
or maybe `=> unknown`?

Reply:
In this case , `unknown` equals to `void`? Both of them looks good for me.